CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.16.0
+
+### Enhancements
+* **Remove ingest implementation**
```

Comment:
Since this is such a big breaking change lets do a more detailed changelog:
- advise on where to look for the new ingest library
- list all extras that are deprecated as a result (so easier for users who have setup to use certain extras to know things are changed)
- high level on what functionalities are removed exactly (e.g., which submodules are now gone)

Reply:
additionally please also add to changelog that the implementation for embed submodule is updated with things like `SecretStr` to represent secrets